### ISSUE TITLE
New rule: Text has minimum contrast

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag-act#475]
The ACT-Rules community group has completed work on _Text has minimum contrast_. We would like to see this rule be added to the WCAG ruleset. You can find the rule here:

**Rule title**: Text has minimum contrast
**Rule id**: afw4f7
**Link to the rule**: https://act-rules.github.io/rules/afw4f7
**License**: W3C Software And Document Notice And License
**Implementors**:
- QualWeb
- SortSite

If you'd like to contact me, please e-mail me at wilcofiers@deque.com. You can also reach me on the W3C Community Slack, my name there is Wilco Fiers.

**Comment by maryjom** [comment]
Link to [survey due 20 August](https://www.w3.org/2002/09/wbs/93339/ACTMinContrast/).

**Comment by maryjom** [comment]
@cpandhi Can you be the liaison for this one? Then open an issue with the survey results and this link to the meeting minutes where the comments were discussed: [Discussion of survey results from the 27 August meeting](https://www.w3.org/2020/08/27-wcag-act-minutes.html#item02)
